web-console/knikubevirt/snapshots/snapshots.md

```diff
@@ -0,0 +1,75 @@
+# Virtual Machines Snapshots
+
+Snapshots allow users to create a copy of the VM disks and memory, mainly for restore and backup purposes. The snapshots are taken live or offline depending on if the VM is running or not. 
+This document describes the snapshot design flow.  
+
+
+
+
+## From VM List 
+
+![Offline VM list - snapshot action from a single VM kebab](img/1-0.jpg)
+When clicking on the kebab menu in the VM list, the user will be able to take a snapshot of the VM, meaning he will copy the disks, memory content, and metadata for restoring, backup, creating templates and cloning VMs.
+
+![Offline Snapshotting status indication](img/1-1.jpg)
+Any Virtual machine that has a snapshot in progress will show this in the state:
+When in this progress mode all actions would be disabled and the “cancel snapshot” would be available.
```

Comment:
nitpick: The text here says "cancel snapshot" and the wireframe shows "stop snapshot".